src/main/java/cat/nyaa/utils/CommandReceiver.java

```diff
@@ -198,7 +200,7 @@ public void acceptCommand(CommandSender sender, Arguments cmd) {
         } catch (NoItemInHandException ex) {
             msg(sender, ex.isOffHand ? "internal.error.no_item_offhand" : "internal.error.no_item_hand");
         } catch (BadCommandException ex) {
-            String msg = ex.getMessage();
+            @LangKey String msg = ex.getMessage();
```

Comment:
Hmm... Seems that I missed something. It
 should complain that ex::getMessage havn't been annotated. It works like a data type here.
I havn't implement check on function return though